blockchain/cancellation_test.go

```diff
@@ -32,7 +32,7 @@ func TestSleepWithContextComplete(t *testing.T) {
 	assert.True(t, completed)
 }
 
-func TestSleepWithContextInterupted(t *testing.T) {
+func TestCancellation_SleepWithContextInterrupted(t *testing.T) {
```

Comment:
is this  `TestCancellation` a special golang test keyword(e.g to skip the test)? Or is just a rename of the tests?

Reply:
Just renaming.